ingeniamc/k2-base-camp
# DR0053A-573 Firmware install both drives simultaneously

### Docs

https://novantamotion.atlassian.net/browse/DR0053A-573

Not everything that was mentioned in the issues was addressed: The texts in the dialog box where adapted to be more in line with MotionLab3, but nothing further was done to make it the same.

The actual change was to merge the two "upload firmware" - buttons into one and removing now superfluous code.

### Test

1. Create a .zfu file with the mappings and firmware files you want to install in the drives.
2. Connect to a setup with two drives in daisy chain.
3. Open K2-Base-Camp and navigate to the "Bootloader" page.
4. Scan for the drives.
5. Upload the .zfu file using the button.
6. Press the "Install" button and confirm with the "Load Firmware" button in the popup-dialog.
7. The firmware should install successfully and a info box should appear that confirms this.
8. Confirm that both drives now have the new firmware installed.
